doc/radosgw/notifications.rst

```diff
@@ -137,6 +137,24 @@ Notification Performance Statistics
     event on each notification, but ``pubsub_push_ok`` and ``pubsub_push_fail``
     are incremented per push action on each notification.
 
+
+Endpoint Configuration Options
+------------------------------
+The following are global configuration options for the different endpoints:
+
+HTTP
+~~~~
+- ``rgw_http_notif_message_timeout``: This is the maximum time in seconds to deliver a notification.
+  Delivery error occurs when the message timeout is exceeded.
+  This value includes the connection time, and hence must be larger than the connection timeout.
+  If set to zero the http client will wait indefinitely. If not set the default will be 10 seconds.
+- ``rgw_http_notif_connection_timeout``: This is the maximum time in seconds to connect to the endpoint.
+  Delivery error occurs when the message timeout is exceeded.
+  If set to zero the defaut value of 300 seconds will be used. If not set the default will be 5 seconds.
+- ``rgw_http_notif_max_inflight``: This is the maximum number of messages in-flight (across all http endpoints).
+  Delivery error (BUSY) occurs when the number of messages is exceeded.
+  If set to zero there is not limit on the number of messages in-flight. If not set the default will be 8192.
```

Comment:
we have a `confval` directive that makes this much easier. for example, `doc/radosgw/config-ref.rst` just has: 
```
.. confval:: rgw_frontends
.. confval:: rgw_data
.. confval:: rgw_enable_apis
```
which renders to https://docs.ceph.com/en/latest/radosgw/config-ref/#ceph-object-gateway-config-reference

this avoids having to duplicate the text between docs and config options